src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -426,6 +426,21 @@ private struct WindowManagerSetStylesParams
 			public string[] Pairs;
 		}
 
+		internal static void SetGeneralStyle(string styleId, string cssText)
```

Comment:
Would be best to use the newer interop mechanisms, same below.

Reply:
These are old parts of the code, no longer needed, so I am removing them 👍